gta/fivem/egg-five-m.json

```diff
@@ -24,7 +24,7 @@
     },
     "variables": [
         {
-            "name": "fivem license",
+            "name": "FiveM License",
```

Comment:
Why upper case this?

Reply:
My bad, probably just my OCD I will removei t.